container/libcontainer/handler.go

```diff
@@ -939,8 +913,5 @@ func newContainerStats(libcontainerStats *libcontainer.Stats, includedMetrics co
 			setCPUSetStats(s, ret)
 		}
 	}
-	if len(libcontainerStats.Interfaces) > 0 {
-		setNetworkStats(libcontainerStats, ret)
```

Comment:
Just to make myself more clear about the removal of `setNetworkStats`.

Here, we believe it is safe to remove the function `setNetworkStats` because:

1. It only appears in this line in the old code, and
2. It actually never be called, because the `libcontainerStats` was created in https://github.com/google/cadvisor/pull/3609/files#diff-28d7cc5aa0eb19db1d3a6d3d16080b4a1940e691ec501ccbd2af273de8034508L92:
```
	libcontainerStats := &libcontainer.Stats{
		CgroupStats: cgroupStats,
	}
```
So the `libcontainerStats.Interfaces` in this line should always be empty, which makes the `if condition` always false. 

Is my understanding correct?

cc: @bobbypage

Reply:
Exactly @cwangVT ! bingo!

Reply:
ack, it makes sense. Thanks @dims for the fix and @cwangVT for reviewing.